soccer/ui/StyleSheetManager.cpp

```diff
@@ -0,0 +1,26 @@
+
+#include "StyleSheetManager.hpp"
+#include <Utils.hpp>
+
+// To add a new style sheet, declare the static variable 
+// in the header file and initialize the path here
+QString StyleSheetManager::DARK = "../soccer/ui/QTDark.stylesheet";
+QString StyleSheetManager::NONE = "";
```

Comment:
Right now, you are storing each stylesheet in its own variable. It's better to put them into a hash table, which makes more logical sense, is cleaner, and is faster.

[Here's an example of using a c++ hashtable](https://en.wikipedia.org/wiki/Unordered_associative_containers_(C%2B%2B)#Usage_example). It might be a little bit unfamiliar, but I can help you understand what's going on!